eq-author-api/tests/utils/contextBuilder/index.js

```diff
@@ -26,78 +25,95 @@ const { createCollapsible } = require("./collapsible");
 
 const buildRouting = require("./buildRouting");
 
+const defaultUser = require("../mockUserPayload");
+
 //@todo - Split into smaller functions to avoid deeply nested chaining
```

Comment:
Can we do this or remove the comment

Reply:
Probably worth doing at some point, it is getting a little complex to figure out what's nested where. Something to look at in the near future?

Reply:
Agreed, but the comment seems pointless. We should open a tech debt issue for it.